lib/inputstreamhelper.py

```diff
@@ -212,6 +212,15 @@ def _run_cmd(self, cmd, sudo=False, ask=True, output=True):
         else:
             return success
 
+    def _load_loop_module(self):
+        """Load loop module."""
+        cmd = ['modprobe', '-q', 'loop']
+        success = self._run_cmd(cmd, sudo=True, ask=True)
+        if success:
+            return True
```

Comment:
The if/else here is superfluous. You can just return `success` directly.

Reply:
Yeah, I was thinking about that, but decided to keep this program structure

Reply:
Removed that superfluous code